src/SFA.DAS.ApplyService.InternalApi/Infrastructure/CharityCommissionApiClient.cs

```diff
@@ -19,14 +19,20 @@ public class CharityCommissionApiClient
         private readonly ILogger<CharityCommissionApiClient> _logger;
         private readonly IApplyConfig _config;
 
+        public CharityCommissionApiClient()
+        {
+
+        }
```

Comment:
Why have these been added?

Reply:
Moq requires a default constructor

Reply:
Shouldn't we be mocking using the interface rather than the concrete class?

Reply:
In this case, the clients are wired up using named HttpClient instances in Startup, not interfaces